unreal_modmetadata/src/lib.rs

```diff
@@ -75,3 +121,92 @@ pub fn from_slice(slice: &[u8]) -> Result<Metadata, Error> {
         _ => Err(Error::unsupported_schema(schema_version)),
     }
 }
+#[cfg(test)]
```

Comment:
Aren't these usually in their own file?

Reply:
can't do that because we are not exporting v1/v2 so they must be in the same file under a different mod